tests/test_crypto/test_cosmos_crypto.py

```diff
@@ -80,7 +80,7 @@ def test_api_none():
 
```

Comment:
Disabled due to consistently failing.

Sould be re-enabled before merging

Reply:
Please enable; this is just an account funding issue; you can fund the account yourself when it happens again (it is now funded) by finding the private key and generating some test wealth for it.